src/foam/dao/history/HistoryRecord.js

```diff
@@ -14,11 +14,15 @@ foam.CLASS({
   tableColumns: [
     'timestamp',
     'objectId',
-    'user',
+    'subject',
```

Comment:
Subject doesn't exist here and isn't being set so this table column will always be missing, let's leave this class the same - please see next comment.

Reply:
yes, this change is wrong. I'll revert this file.

Reply:
updated.